src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelListItem.test.tsx

```diff
@@ -135,7 +135,14 @@ describe('<TrackPanelListItem />', () => {
     });
 
     it('toggles the expanded/collapsed state of the track when clicked on the expand button', async () => {
-      const { container } = wrapInRedux();
+      store = mockStore(mockState);
+      const { container } = render(
+        <Provider store={store}>
+          <TrackPanelListItem {...defaultProps}>
+            <TrackPanelListItem {...defaultProps} />
+          </TrackPanelListItem>
```

Comment:
`TrackPanelListItem` nested within `TrackPanelListItem` doesn't look right here. Please check.

Reply:
Yep, @imran is right; this is how this PR is handling transcript tracks inside of the gene track.